frontend/components/navbar/index.tsx

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import { NextComponentType } from "next";
 import Link from "next/link";
 import { signIn, signOut, useSession } from "next-auth/react";
-import { Box, Stack, Link as _Link, Button, IconButton, useColorMode } from "@chakra-ui/core";
+import { Box, Stack, Link as ChakraLink, Button, IconButton, useColorMode } from "@chakra-ui/core";
```

Comment:
@FaithDaka - I made a small change to this PR to change the chakra link import from `_Link` to `ChakraLink`, just to make it a bit more explicit what we are referring to